views/index.ejs

```diff
@@ -11,7 +11,9 @@
     <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
     <ul>
       <!-- List the names of the items in your cart here -->
-
+        <% cart.forEach(function(product){ %>
+               <li><%= product.name %></li>
+        <% }) %>
```

Comment:
You've got your forEach loop set up correctly and it is printing the product's name correctly, but you are missing the form needed in order to actually submit and make a post request. Inside that form you will also need some way to access the product's id so your post route has the ability to get the id from req.body.id. Since you don't necessarily want the id to print by each cookie (that would look funny to a user since the ID number doesn't mean anything to a shopper), you can have a hidden input with the value set to <%= product.id %>.